main.tf

```diff
@@ -1,7 +1,7 @@
 module "tags" {
   source = "git::https://github.com/rhythmictech/terraform-terraform-tags.git?ref=v0.0.2"
```

Comment:
can you bump to v1.1 to address the superluous tags issue?

Reply:
Sure thing